src/ui/src/components/forms/IntegrationForm/index.tsx

```diff
@@ -86,8 +86,8 @@ const IntegrationForm: React.FC = () => {
       <EuiFormRow
         label="Webhook URL"
         helpText={
-          <EuiLink>
-            <span>Create a Slack Webhook URL</span>
+          <EuiLink href={'https://support.smrtbeat.com/hc/en-us/articles/206429576-How-do-I-get-the-Webhook-URL-to-use-for-Slack-notifications-'} target="_blank">
```

Comment:
https://api.slack.com/messaging/webhooks

Reply:
Changed. The reason included a third party link is because IMO the way it proposes to get a channel's webhook is easier than Slack's official docs, but if you prefer official docs we can do that.